shacl12-core/index.html

```diff
@@ -4721,12 +4743,19 @@ <h2>Acknowledgements</h2>
 		<section class="appendix informative">
 		  <h2>Revision History</h2>
 			<p>
-				The detailed list of changes and their diffs can be found in the Git repository.
+				The detailed list of changes and their diffs can be found in the <a href="https://github.com/w3c/data-shapes">Git repository</a>.
 			</p>
 			<ul>
 				<li><b>2024-02-14</b>: New work started by cloning the main SHACL spec and splitting it into SHACL Core and SHACL-SPARQL</li>
 			</ul>
 		</section>
+
+		<section class="appendix informative" id="changes-12">
+			<h2>Changes between SHACL 1.0 Core and SHACL 1.2 Core</h2>
+			<ul>
+				<li>Added the new class <a href="#ShapeClass"><code>sh:ShapeClass</code></a> as syntactic sugar for implicit class targets, see <a href="https://github.com/w3c/data-shapes/issues/212">Issue 212</a></li>
```

Comment:
It has behavior so "syntactic sugar"
```suggestion
				<li>Added the new class <a href="#ShapeClass"><code>sh:ShapeClass</code></a> for implicit class targets, see <a href="https://github.com/w3c/data-shapes/issues/212">Issue 212</a></li>
```